src/System.Private.Uri/src/System/UriHelper.cs

```diff
@@ -634,6 +658,13 @@ internal static void EscapeAsciiChar(char ch, char[] to, ref int pos)
             to[pos++] = s_hexUpperChars[ch & 0xf];
         }
```

Comment:
Is this char[]-based overload still used?

Reply:
No, they are not. Removed.